tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/HttpMessageHandlerTests.cs

```diff
@@ -158,7 +160,8 @@ public HttpMessageHandlerTests(ITestOutputHelper output)
                 else
                 {
                     // assert that "_dd.p.tid" was NOT added
-                    Assert.Null(traceIdUpperTagFromHeader);
+                    // NOTE that since swapping to Activity for the sample, we'll have 128bit traceIDs always
```

Comment:
I'm not sure sure on removing this `Assert.Null(traceIdUpperTagFromHeader);`, so any suggestions welcome